app/controllers/frames_controller.rb

```diff
@@ -116,7 +116,7 @@ def scoped_frames
   end
 
   def set_frame
-    authorize! @frame = scoped_frames.friendly.find(params[:id].to_s.downcase)
+    authorize! @frame = Frame.all.friendly.find(params[:id].to_s.downcase)
```

Comment:
```suggestion
    authorize! @frame = Frame.friendly.find(params[:id].to_s.downcase)
```

`.all` is not necessary I think